crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -604,14 +640,42 @@ impl<'w, T: Fetch<'w>> Fetch<'w> for OptionFetch<T> {
     }
 }
 
-/// Change trackers for component `T`
+/// [`WorldQuery`] that tracks changes and additions for component `T`.
+///
+/// Wraps a [`Component`] to track whether the component changed for the corresponding entities in
+/// a query since the last time the component has been observed.
```

Comment:
N
```suggestion
/// a query since the last time the system that includes these query ran.
```

Reply:
I'm not *convinced* this is general enough for the direct world access use case.